bot_commands.py

```diff
@@ -92,18 +95,35 @@ def main(bot, update):
         text=rest('Mainchain Stage')
     )
 
+
 def side(bot, update):
     bot.send_message(
         chat_id=update.message.chat_id,
         text=rest('Sidechain Stage')
     )
 
-def build(bot, update):
+
+def workshop_rinkeby(bot, update):
     bot.send_message(
         chat_id=update.message.chat_id,
-        text=rest('Buidl Room')
+        text=rest('rinkeby room')
```

Comment:
This probably needs to be capitalized.

Reply:
Ok added! and capitalized the other areas this was lowercase as well.